src/components/BlogPage/BlogListItem.js

```diff
@@ -1,35 +1,64 @@
 import React from 'react';
 import PropTypes from 'prop-types';
-import { Typography, Link } from '@material-ui/core';
+import { Typography, Card, CardContent, CardActionArea, withStyles, Divider } from '@material-ui/core';
 
 import LinkNoStyle from '../LinkNoStyle/LinkNoStyle';
 
+const styles = theme => ({
+	container: {
+		marginTop: theme.spacing(1),
```

Comment:
as shorthand, I'm pretty sure you can do `margin: theme.spacing(1) 0,` instead of specifically naming `marginTop` and `marginBottom`

Reply:
ooo vv good to know tyty